src/main/java/fr/moribus/imageonmap/commands/maptool/GetCommand.java

```diff
@@ -49,11 +53,53 @@
 public class GetCommand extends IoMCommand {
     @Override
     protected void run() throws CommandException {
-        Player player = playerSender();
-        if (getMapFromArgs().give(player)) {
-            info(I.t("The requested map was too big to fit in your inventory."));
-            info(I.t("Use '/maptool getremaining' to get the remaining maps."));
+        ArrayList<String> arguments = getArgs();
+
+        if (arguments.size() > 2) {
+            warning(I.t("Too many parameters! Usage: /maptool get [playername]:<mapname>"));
+            return;
+        }
+        if (arguments.size() < 1) {
+            warning(I.t("Too few parameters! Usage: /maptool get [playername]:<mapname>"));
+            return;
         }
+        final String playerName;
+        final String mapName;
+        final Player sender = playerSender();
+
+        if (arguments.size() == 1) {
+            playerName = sender.getName();
+            mapName = arguments.get(0);
+        } else {
+            if (!Permissions.GETOTHER.grantedTo(sender)) {
+                info(sender, I.t("You can't use this command"));
+                return;
+            }
+            playerName = arguments.get(0);
+            mapName = arguments.get(1);
+        }
+
+
+
+
+        //TODO passer en static
+        ImageOnMap.getPlugin().getCommandWorker().offlineNameFetch(playerName, uuid -> {
```

Comment:
Sait-on jamais qu'iel se soit déconnecté⋅e entre temps
```suggestion
        ImageOnMap.getPlugin().getCommandWorker().offlineNameFetch(playerName, uuid -> {
            if (!sender.isOnline()) return;
```